packages/atomic/src/components/atomic-result-list/atomic-result-list.tsx

```diff
@@ -121,18 +122,123 @@ export class AtomicResultList implements InitializableComponent {
       });
   }
 
-  private get results() {
-    return this.resultListState.results.map((result) => (
+  private getTemplate(result: Result) {
+    return this.resultTemplatesManager.selectTemplate(result) || '';
+  }
+
+  private buildResult(result: Result) {
+    return (
       <atomic-result
         key={`${result.raw.permanentid}${this.resultListState.searchUid}`}
         result={result}
         engine={this.bindings.engine}
         display={this.display}
         density={this.density}
         image={this.image}
-        content={this.resultTemplatesManager.selectTemplate(result) || ''}
+        content={this.getTemplate(result)}
       ></atomic-result>
-    ));
+    );
+  }
+
+  private buildListResults() {
+    return this.resultListState.results.map((result) =>
+      this.buildResult(result)
+    );
+  }
+
+  private buildTableResults() {
+    const fieldColumns = Array.from(
+      parseHTML(
+        this.getTemplate(this.resultListState.results[0])
+      ).querySelectorAll('atomic-table-element')
+    );
+
+    return (
+      <table>
+        <thead>
+          <tr>
+            <th>Description</th>
+            {fieldColumns.map((column) => (
+              <th>{column.getAttribute('label')}</th>
+            ))}
+          </tr>
+        </thead>
+        <tbody>
+          {this.resultListState.results.map((result) => (
+            <tr>
+              <td>{this.buildResult(result)}</td>
```

Comment:
Erhm... what's the goal of always loading a template named "description" in the first section of the table ? 

Why is it completely different from others columns ?

IMO, all columns should have the same logic. Instead of forcing this, what should be done is this if the user wants to see a "Description" column:

```
<atomic-table-element label="Description">
  [... the content I want for my description ..., most probably just the excerpt]
</atomic-table-element>
```

Reply:
The reason for that was to use the same template definition used for the list. I'm not opposed to using a separate template definition though.